src/main/java/org/example/Calculator.java

```diff
@@ -0,0 +1,48 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class Calculator {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter operand1: ");
+        double operand1 = scanner.nextDouble();
+
+        System.out.print("Enter operator (+, -, x, /): ");
+        String operator = scanner.next();
+
+        System.out.print("Enter operand2: ");
+        double operand2 = scanner.nextDouble();
+
+        double result = 0;
+
+        switch (operator) {
+            case "+":
+                result = operand1 + operand2;
+                break;
+            case "-":
+                result = operand1 - operand2;
+                break;
+            case "%":
+                result = operand1 * operand2;
+                break;
+            case "/":
+                if (operand2 != 0) {
+                    result = operand1 / operand2;
+                } else {
+                    System.out.println("Error: Division by zero is not allowed.");
```

Comment:
It's better to use System.err.println() for print errors (if you don't throw an exception)